server/controllers/calendarController.test.ts

```diff
@@ -1,7 +1,7 @@
 import { Request, Response } from 'express'
 import CalendarController from './calendarController'
 import mfaBannerType from '../helpers/mfaBannerType'
-import { CalendarService, NotificationCookieService, UserAuthenticationService, UserService } from '../services'
```

Comment:
I've left the service in for now, will remove in next PR as will also remove the db stuff too